### PR TITLE
Update components in Studio Pro

### PR DESCRIPTION
**Pull request by Luyao-Zhang-1**

### CODE REVIEW

File: content/appstore/general/app-store-content.md [hunk 1]
```diff
@@ -98,7 +98,29 @@ To add the Rating widget from the Marketplace via Studio Pro, follow these steps
...
+
+4. Click the component or **Read more** to view the details of the component.
+
+5. Go to the **All Versions** tab.
+
+   ![all-versions-tab](attachments/app-store-content/all-versions-tab.png)
+
+6. Click **Download** to download the latest version. The **Question** dialog box opens and asks if you want to overwrite the existing package.
+
+7. Click **Yes** to proceed. The widget is updated to the latest version.
```
Comment (MariaShaposhnikova) [inline content/appstore/general/app-store-content.md:121]: I would state the result (that the widget is now updated) as a separate sentence and not in the list. For example, like in this document >2.2 section: https://docs.mendix.com/studio/page-editor#creating-new-page

File: content/appstore/general/app-store-content.md [hunk 2]
```diff
@@ -155,7 +177,31 @@ Unlike a widget, you can see a module in your App Explorer. A module contains a
...
+
+6. Click **Download** to download the latest version. The **Import Module** dialog box opens.
+
+   ![Import Module dialog box](attachments/app-store-content/import-module.png)
+
+7. Select **Replace existing module** and click **Import**.
+
+8. When the **Warning** dialog box shows the files that will be overwritten, click **OK**.
+
+9. Wait until a pop-up window states that the module was successfully imported. Click **OK**. The module is updated to the latest version.
```
Comment (MariaShaposhnikova) [inline content/appstore/general/app-store-content.md:202]: I would state the result (that the module is now updated) as a separate sentence and not in the list.

**Review by MariaShaposhnikova — COMMENTED** [review]
Just two small comments, the rest looks great!

File: content/appstore/general/app-store-content.md (hunk 1 above)
Comment (Luyao-Zhang-1) [reply content/appstore/general/app-store-content.md]: That's better. I changed the text.

File: content/appstore/general/app-store-content.md (hunk 2 above)
Comment (Luyao-Zhang-1) [reply content/appstore/general/app-store-content.md]: That's better. I changed the text.

**Pull request merged**